core/src/main/java/org/axonframework/queryhandling/UpdateHandler.java

```diff
@@ -16,5 +16,39 @@
 
 package org.axonframework.queryhandling;
 
+/**
+ * Defines update handler callbacks for subscription query.
```

Comment:
The javadoc should indicate that the `onInitialResult` is always invoked _before_ any `onUpdate` calls.